src/drivers/actuators/voxl_esc/voxl_esc.cpp

```diff
@@ -1195,6 +1202,12 @@ bool VoxlEsc::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
 	//in Run() we call _mixing_output.update(), which calls MixingOutput::limitAndUpdateOutputs which calls _interface.updateOutputs (this function)
 	//So, if Run() is blocked by a custom command, this function will not be called until Run is running again
 
+	// counter to track how many times uart_write has been called with gpio data
+	static int gpio_write_counter = 0;
```

Comment:
This is kind of nitpicky because I can't imagine someone would ever have multiple instances of this driver, but can you avoid the static variables?

Reply:
@dagar Made the two static local variables into member variables.